scripts/build.ts

```
@@ -69,6 +69,17 @@ for (const targetFormat of ["cjs", "esm"] as const) {
             break;
         case "esm":
             run(`npx tsc --module es2020 --outDir ${distDirPath}`);
+            const svelteFiles = fs
+                .readdirSync(pathJoin(projectDirPath, "src", "svelte"), { withFileTypes: true })
+                .filter(
+                    f => f.isFile() && (f.name.endsWith(".svelte") || f.name.endsWith(".svelte.d.ts"))
+                );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)